app/controllers/extension_settings_controller.rb

```diff
@@ -3,5 +3,7 @@ def mute
     settings = ExtensionSetting.find(params[:id])
     settings.mute_emails = true
     settings.save
+
+    render plain: 'Emails successfully muted'
```

Comment:
Let's go with "This email has been muted. To unmute, reinstall this extension."

Reply:
Should this be a `JSON` response?

Reply:
it's not part of the API to be `JSON` - users get in here from email links